lib/watch-sidebar.js

```diff
@@ -102,17 +102,41 @@ export default class WatchSidebar {
   }
 
   removeWatch() {
-    const watches = this.watchViews.map((v, k) => ({
-      name: v.getCode(),
-      value: k
-    }));
-    WatchesPicker.onConfirmed = item => {
-      this.watchViews[item.value].destroy();
-      this.watchViews.splice(item.value, 1);
-      if (this.watchViews.length === 0) this.addWatch();
-    };
-    WatchesPicker.setItems(watches);
-    WatchesPicker.toggle();
+    const watches = this.watchViews
+      .map((v, k) => ({
+        name: v.getCode(),
+        value: k
+      }))
+      .filter(obj => obj.value !== 0 || obj.name !== "");
+
+    watchesPicker = new SelectListView({
```

Comment:
That's interesting. I would expect this to fail, maybe babel fixes this for us. Anyway, I think we should use
```js
const watchesPicker = new SelectListView({
```